simple_locations/json_views.py

```diff
@@ -0,0 +1,13 @@
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from django.utils.safestring import mark_safe
+from django.http import JsonResponse
+import json
+
+from .models import Area
+
+def area(request, area_id):
+    return JsonResponse(Area.geojson(area_id))
+
+def children(request, area_id):
+    return JsonResponse(Area.geojson(area_id, children=True))
```

Comment:
May as well remove the optional argument `children` in `geojson` and call `geojson_children` from here, since the two class methods are independent.